flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/JobExceptionsInfoWithHistory.java

```diff
@@ -208,7 +214,34 @@ public static class ExceptionInfo {
         private final Map<String, String> failureLabels;
 
         public ExceptionInfo(String exceptionName, String stacktrace, long timestamp) {
-            this(exceptionName, stacktrace, timestamp, Collections.emptyMap(), null, null, null);
+            this(
+                    exceptionName,
+                    stacktrace,
+                    timestamp,
+                    Collections.emptyMap(),
+                    null,
+                    null,
+                    null,
+                    null);
+        }
+
+        public ExceptionInfo(
```

Comment:
This constructor is used by `JobExceptionsHandler` and unit tests. Can we change these to use the next constructor so we can delete this one?